include/allegro5/platform/aintwin.h

```diff
@@ -92,6 +92,9 @@ struct ALLEGRO_DISPLAY_WIN
    int toggle_w;
    int toggle_h;
 
+   //allegro don't manager the display
+   bool aldontmanagerdisplay;
```

Comment:
Could you rename this to `dont_manage_display` or alternatively, flip call it `manage_display` and flip all the conditions in the code?